README.md

```diff
@@ -9,6 +9,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
 | [PyCon India](https://in.pycon.org/2019/) |  12-15 Oct |  Chennai Trade Center and IIT Madras Research Park  | India's premier conference on using and developing the Python programming language | No |
+| [PyData India](https://pydata.org/delhi2019/) | 3-4 Aug |  IIIT Delhi | A conference for data-science enthusiasts for using Python programming language to work with data | No | 
```

Comment:
Thanks @Naba7 :)

This list is in the reverse chronological order, can you move this between DevConf and JSFoo: Vueday? Looks good otherwise. :+1: